index.js

```diff
@@ -148,6 +151,7 @@ ZongJi.prototype._fetchTableInfo = function(tableMapEvent, next) {
     tableMapEvent.schemaName, tableMapEvent.tableName);
 
   this.ctrlConnection.query(sql, function(err, rows) {
+    // TODO: Remove throws to make the module safe
```

Comment:
This one can happen multiple times (gets ran for each `TableMap` packet) and can be emitted from the `ZongJi` object.